### PR TITLE
Style: Revert uncrustify for portable directories

### PR DESCRIPTION
**Pull request by alfred2g**
Revert uncrustify for untested portable directories

Description
-----------
Uncrustify messed up some inline assembly for some compilers
We are reverting some compiler directories inside portable

Test Steps
-----------
already changed folders/files after previous uncrustify

   portable/ARMv8M/non_secure/port.c
   portable/ARMv8M/non_secure_port.c

   portable/GCC/ARM_CM23/non_secure/port.c
   portable/GCC/ARM_CM23_NTZ/non_secure/port.c
   portable/GCC/ARM_CM33/non_secure/port.c
   portable/GCC/ARM_CM33_NTZ/non_secure/port.c
   portable/GCC/ARM_CM4_MPU/port.c
   portable/GCC/ARM_CM4_MPU/portmacro.h
   portable/GCC/ARM_CM0/port.c
   portable/GCC/ARM_CM3/port.c
   portable/GCC/ARM_CM3_MPU/port.c
   portable/GCC/ARM_CM4F/port.c
   portable/GCC/ARM_CM7/r0p1/port.c
   portable/GCC/AVR_AVRDx/port.c
   portable/GCC/AVR_AVRDx/porthardware.h
   portable/GCC/AVR_AVRDx/portmacro.h
   portable/GCC/AVR_Mega0/port.c
   portable/GCC/AVR_Mega0/porthardware.h
   portable/GCC/AVR_Mega0/portmacro.h
   portable/GCC/ARM_CR5/port.c

   portable/ThirdParty/GCC/ARC_EM_HS/freertos_tls.c
   portable/ThirdParty/GCC/ARC_EM_HS/port.c
   portable/ThirdParty/GCC/ARC_EM_HS/portmacro.h
   portable/ThirdParty/GCC/Xtensa_ESP32/include/portmacro.h
   portable/ThirdParty/GCC/Xtensa_ESP32/port.c
   portable/ThirdParty/GCC/Xtensa_ESP32/portasm.S
   portable/ThirdParty/GCC/Xtensa_ESP32/xtensa_init.c
   portable/ThirdParty/GCC/Xtensa_ESP32/xtensa_intr.c
   portable/ThirdParty/GCC/Xtensa_ESP32/xtensa_vector_defaults.S
   portable/ThirdParty/GCC/Xtensa_ESP32/xtensa_vectors.S

   portable/IAR/ARM_CM23/non_secure/port.c
   portable/IAR/ARM_CM23_NTZ/non_secure/port.c
   portable/IAR/ARM_CM33/non_secure/port.c
   portable/IAR/ARM_CM33_NTZ/non_secure/port.c
                                                                                                                                                                                     
   portable/IAR/ARM_CM4F_MPU/port.c
   portable/IAR/ARM_CM4F_MPU/portasm.s
   portable/IAR/ARM_CM4F_MPU/portmacro.h

   portable/IAR/ARM_CM0/port.c

   portable/IAR/ARM_CM3/port.c
   portable/IAR/ARM_CM4F/port.c

   portable/IAR/ARM_CM7/r0p1/port.c

   portable/IAR/AVR_AVRDx/port.c
   portable/IAR/AVR_AVRDx/porthardware.h
   portable/IAR/AVR_AVRDx/portmacro.h
   portable/IAR/AVR_AVRDx/portmacro.s90
   portable/IAR/AVR_Mega0/port.c
   portable/IAR/AVR_Mega0/porthardware.h
   portable/IAR/AVR_Mega0/portmacro.h
   portable/IAR/AVR_Mega0/portmacro.s90

   portable/RVDS/ARM_CM0/port.c
   portable/RVDS/ARM_CM3/port.c
   portable/RVDS/ARM_CM4F/port.c
   portable/RVDS/ARM_CM7/r0p1/port.c
   portable/RVDS/ARM_CM4_MPU/port.c
   portable/RVDS/ARM_CM4_MPU/portmacro.h

   portable/MemMang/heap_4.c

   portable/CCS/ARM_CM3/port.c
   portable/CCS/ARM_CM4F/port.c

   portable/MikroC/ARM_CM4F/port.c

   portable/Tasking/ARM_CM4F/port.c

   portable/Common/mpu_wrappers.c

Related Issue
-----------
#90, #91, #107, #111

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: portable/BCC/16BitDOS/Flsh186/prtmacro.h [hunk 1]
```diff
@@ -39,59 +39,59 @@
...
-#define portSHORT         int
-#define portSTACK_TYPE    uint16_t
-#define portBASE_TYPE     portSHORT
+#define portCHAR		char
+#define portFLOAT		float
+#define portDOUBLE		long
+#define portLONG		long
+#define portSHORT		int
+#define portSTACK_TYPE	uint16_t
+#define portBASE_TYPE	portSHORT
```
Comment (n9wxu) [inline portable/BCC/16BitDOS/Flsh186/prtmacro.h:48]: This is wierd.  What happened?

Comment (alfred2g) [reply portable/BCC/16BitDOS/Flsh186/prtmacro.h:48]: this is a revert.. back to tabs

Comment (alfred2g) [reply portable/BCC/16BitDOS/Flsh186/prtmacro.h:48]: This is a revert.. back to tabs

**Pull request merged**